lib/string-checker.js

```diff
@@ -83,17 +83,6 @@ StringChecker.prototype = {
      * @throws Error on invalid source code.
      */
     _parse: function(str) {
```

Comment:
The problem is that StringChecker catches parse errors. If we would parse in JsFile constructor, we won't be able to create new JsFile instance. Currently JsFile is created with empty esprima node tree information.